### PR TITLE
Added a boilerplate Code of Conduct

### PR DESCRIPTION
**Pull request by dmcalester**
This is a boilerplate Code of Conduct. We may choose to add to, replace or amend this document as we listen and learn to our  internal and external communities.

Other Code’s of Conduct we may be interested in

https://18f.gsa.gov/code-of-conduct/
https://github.com/codeforamerica/codeofconduct
https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at uxsupport@rocketcom.com. All
+complaints will be reviewed and investigated and will result in a response that
```
Comment (sknep) [inline CODE_OF_CONDUCT.md:59]: promptly?

**Comment by sknep** [comment]
This adds it to the website repo, that is, astrouxds.com -- would you like to also add it to astro-components?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rocketjeff) [reply CODE_OF_CONDUCT.md:59]: Don't promise what we can't deliver. We don't have dedicated staff or a tested workflow for these complaint reviews.

Comment (dmcalester) [reply CODE_OF_CONDUCT.md:59]: Good point @rocketjeff I’ll remove for now

**Pull request merged**